syncer/syncer_test.go

```diff
@@ -130,31 +130,16 @@ func TestStartAndShutdown(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.TODO())
 	ts := newTestSyncer(ctx, t, time.Millisecond*5)
 
-	syncedCh := ts.syncer.RegisterForSynced(context.TODO())
-	atxSynchedCh := ts.syncer.RegisterForATXSynced()
-
 	require.False(t, ts.syncer.IsSynced(context.TODO()))
 	require.False(t, ts.syncer.ListenToATXGossip())
 	require.False(t, ts.syncer.ListenToGossip())
 
 	// the node is synced when current layer is <= 1
 	ts.syncer.Start(context.TODO())
 
-	select {
-	case <-syncedCh:
-	case <-time.After(1 * time.Second):
-		require.Fail(t, "node should be synced")
-	}
-
-	select {
-	case <-atxSynchedCh:
-	case <-time.After(1 * time.Second):
-		require.Fail(t, "node should be atx synced")
-	}
-
-	require.True(t, ts.syncer.IsSynced(context.TODO()))
-	require.True(t, ts.syncer.ListenToATXGossip())
-	require.True(t, ts.syncer.ListenToGossip())
```

Comment:
Why where these asserts removed? Shouldn't they still be part of the test even after the removal of `RegisterForSynced`?

Reply:
because it's ensured in line 141 now.